README.md

```diff
@@ -1,26 +1,36 @@
 # Chimera
-Chimera is the update to Halo: Combat Evolved for the PC that we should have had
-but never got.
+
+Chimera is the update for the original releases of Halo: Combat Evolved on PC that we should've had but never gotten.
 
 It is licensed under version 3.0 of the GNU General Public License. See LICENSE
 for more information.
 
 The official repository is hosted at https://github.com/SnowyMouse/chimera
 
-
-
-
 ## Table of contents
+
+- [Game version support](#game-version-support)
 - [Installation](#installation)
 - [Mod support](#mod-support)
 - [Features](#features)
 - [FAQ](#faq)
 
+## Game version support
 
+Currently Chimera supports the following Halo PC releases (Xbox releases aren't supported):
 
-
+| Name                              | Code Name | Support (Version)  |
+| --------------------------------- | --------- | :----------------: |
+| Halo: Combat Evolved              | halor     | ✅ (01.00.10.0621) |
+| Halo Custom Edition               | halom     | ✅ (01.00.10.0621) |
+| Halo Trial                        | halod     | ✅ (01.00.00.0578) |
+| Halo Beta                         | halo      |         ❌         |
+| Halo Mac                          | halomac   |         ❌         |
+| Halo Trial (Mac)                  | halomacd  |         ❌         |
```

Comment:
I thought this was typically shortened to 'halomd'.

Reply:
Ah, my bad. [HaloMD refers to a mod for Halo Mac Demo](https://www.halomd.net/).

Reply:
Yes, i'm using the code names as how the games were registered in the gamespy master server
halor = Halo Retail (Combat evolved)
halom = Halo Multiplayer Expansion (Custom edition)
halod = Halo Demo (Trial)
halo = Halo Beta